app/plugin.go

```diff
@@ -330,6 +320,15 @@ func (ch *Channels) syncPlugins() *model.AppError {
 					mlog.Error("Failed to remove local installation of managed plugin before sync", mlog.String("plugin_id", pluginID), mlog.Err(err))
 				}
 			}
+
+			// Disable focalboard in product mode after it is removed from PluginSettings.Directory
```

Comment:
This block is about purging existing plugins before unpacking new ones -- would it make sense to do this check in the block on line 343 below instead? And instead of disabling, simply skip installing the plugin in the first place?